contrib/fonttools/Makefile.am

```diff
@@ -35,6 +35,8 @@ noinst_PROGRAMS = findtable pfadecrypt rmligamarks stripttc woff	\
 
 woff_LDADD = $(ZLIB_LIBS)
 dewoff_LDADD = $(ZLIB_LIBS)
+ttf2eps_LDADD = $(top_builddir)/Unicode/libgunicode.la \
```

Comment:
Seems a bit heavy to add a dependency on these dlls for just the one function (and considering the standalone nature of the rest of the contrib tools)

Reply:
That dependency isn't already there only because there is a lot of code copied and pasted in `ttf2eps`. :) Same for `showttf`

We really should go though it and remove all the duplicated and outdated code. This would leave only the core ttf2eps logic there. That is the point of a `libfontforge`, isn't it? ;)

Reply:
True, but I can also see the reasoning; they have no dependencies on anything else, so they can be shipped independently of FF itself. 

For example, if I'm reading this right, the [fontforge-extras](https://packages.debian.org/sid/fontforge-extras) package ships with no dependency on fontforge itself.

Reply:
I forgot to comment here, but our repackaging dropped fontforge-extras. I'm hoping that Debian will follow suit.